specs/operator/rbac.spec

```diff
@@ -14,6 +14,8 @@ Importance: High
 This scenario helps you to disable creation of RBAC resources at cluster level.
 
 Steps:
+  * Update TektonConfig CR to use param with name createRbacResource and value "true" to "enable" auto creation of RBAC resources
```

Comment:
By default `createRbacResource` is enbled so do we need to add this step :thinking:

Reply:
If `createRbacResource` is `enabled` by default , we don't need this.

Reply:
Yup we can remove then

Reply:
If I use a cluster on which the `createRbacResource` is set to `false`, the test fails, so I am just setting this as a pre condition